src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -0,0 +1,137 @@
+@namespace("org.ga4gh.models")
+/**
+This protocol defines annotations on GA4GH genomic sequences It includes two
+types of annotations: continuous and discrete hierarchical.
+
+The discrete hierarchical annotations are derived from the Sequence Ontology
+(SO) and GFF3 work 
+
+   http://www.sequenceontology.org/gff3.shtml
+
+The goal is to be able to store annotations using the GFF3 and SO conceptual
+model, although there is not necessarly a one-to-one mapping in Avro records
+to GFF3 records.
+
+The minimum requirement is to be able to accurately represent the current
+state of the art annotation data and the full SO model.  Feature is the
+core generic record which corresponds to the a GFF3 record.
+*/
+protocol SequenceAnnotations {
+
+  import idl "common.avdl";
+  import idl "metadata.avdl";
+
+  /**
+  Type defining a collection of attributes associated with various protocol
+  records.  Each attribute is a name that maps to an array of one or more
+  values.  Values can be strings, external identifiers, or ontology terms.
+  Values should be split into the array elements instead of using a separator
+  syntax that needs to parsed.
+  */
+  // TODO: how are multiple instances of a given attribute vs multiple values
+  // for an attribute distinguished
+  record Attributes {
```

Comment:
My initial impression is that this data type is convoluted; by default, I would assume this type of field should follow the same pattern we have everywhere else in the API i.e. `map<[]string>`. In particular, referencing an `ExternalIdentifier` does not seem like a problem unique to sequence annotations. I'd guess this is probably meant to emulate the behavior of GFF3, but given the current documentation I can't tell if this is capturing some semantic nuance in the spec, or if it's overfitting and this is an artifact of it being a file format.

This could continue to follow the normal API patterns we have elsewhere by splitting this into three fields on the `FeatureSet`:

```
map<array<string>> info;
array<ExternalIdentifier> externalIds;
map<array<OntologyTerm>> ___;
```

I don't understand the last use case well enough to propose a name for that field.